### DISCUSSION TITLE
To avoid "Please enable browser cookies to view the page content." problem in cypress test

### DISCUSSION THREAD

**Discussion by knitehias** [cypress-io/cypress#15469] [Questions and Help]
Hello Experts,

When I am using cypress to test our partner's website, cypress stopped and showing this "Please enable browser cookies to view the page content."

After inspect, it is due to our partner is setting some cookies dynamically and somehow our cypress tests are not doing that set somehow. Those cookies include but does not limit with "Path=/; Domain=.domain.com; Secure; HttpOnly"

May I ask how to fix or workaround such issues, or how to enable the page to set cookies on cypress test?

**Comment by sowmyasridharamurthy** [comment]
I am facing the same issue, its surpising to see no reply till now. 

In my case the cookies are not set propely and due to which the redirections are not happening and i  receive session expired message.

**Comment by sowmyasridharamurthy** [comment]
@bahmutov can we get some attention here please

**Reply by bahmutov** [reply]
I am sure the Cypress team would address this issue if they had a reproducible example

**Reply by nhariom2000** [reply]
> I am sure the Cypress team would address this issue if they had a reproducible example

@bahmutov I have done this code in github for accessing my SSO profile and getting a detail related to my project there but there is an an error saying currently all the cookies are blocked in my browser . Anyone know Any answers?


describe("gitLogin with microsoft SSO", () => {
it("gitLogin with microsoft", () => {
cy.visit("https://github.com/login");
cy.get("[name='login']").type("user@web.net");
cy.get("[name='password']").type("password123");
cy.get("[name='commit']").click();
cy.get("[data-target='webauthn-get.button']").click();
cy.get("[class='dropdown-caret']").last().click();
cy.wait(3000);
cy.get("span:contains(projectName)").last().click();
cy.get("[type='submit']").click();
cy.origin("https://login.microsoftonline.com/", {
args: {
username: "user@web.net",
password: "password123",
ssoUsernameBox: "[name='loginfmt']",
nextButton: "[id='idSIButton9']",
ssoPasswordBox: "[name='passwd']",
},
},
({ username, password, ssoUsernameBox, ssoPasswordBox, nextButton }) => {
cy.log(username);
const uName = username;
cy.get(ssoUsernameBox).clear().type(uName);
cy.get(nextButton).click();
cy.wait(3000);
cy.get(ssoPasswordBox).type(password, { log: false }).should(el$ => {
if (el$.val() !== password) {
throw new Error('Different value of typed password');
}
});
cy.get(nextButton).click();
});
});
});


this should be reproducible right ... if you need username and password too pls reply to this thread ... but this is the general login of github using microsoft authenticator

![Screenshot (785)](https://user-images.githubusercontent.com/111478262/202438040-c37b2c8e-c4a4-4fea-929a-fee23e8a1e22.png)

**Comment by jlbizet** [comment]
I have the same issue. Im trying everything and i cant fixed 
![image](https://user-images.githubusercontent.com/78700173/141844243-5f70433f-28c2-41b4-8088-ed55ebf6c20c.png)
